wp-content/themes/theme_name/functions.php

```diff
@@ -1,20 +1,13 @@
 <?php
 
-/**
- * After setup theme hook
- */
+require get_parent_theme_file_path( '/inc/admin.php' );
+require get_parent_theme_file_path( '/inc/enqueue.php' );
+require get_parent_theme_file_path( '/inc/general.php' );
+require get_parent_theme_file_path( '/inc/helpers.php' );
+require get_parent_theme_file_path( '/inc/media.php' );
+require get_parent_theme_file_path( '/inc/menu.php' );
+require get_parent_theme_file_path( '/inc/pagination.php' );
+require get_parent_theme_file_path( '/inc/theme-options.php' );
+require get_parent_theme_file_path( '/inc/widgets.php' );
+require get_parent_theme_file_path( '/inc/xmlrpc.php' );
 
```

Comment:
The best practices are to hook all the 'includes' file to `after_setup_theme` hook.